gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/uikit/internal/registerSimulatorTasks.kt

```diff
@@ -6,24 +6,34 @@
 package org.jetbrains.compose.experimental.uikit.internal
 
 import org.gradle.api.*
+import org.gradle.api.tasks.TaskProvider
 import org.jetbrains.compose.desktop.application.internal.Arch
 import org.jetbrains.compose.desktop.application.internal.MacUtils
 import org.jetbrains.compose.desktop.application.internal.currentArch
 import org.jetbrains.compose.experimental.dsl.DeployTarget
+import org.jetbrains.compose.experimental.dsl.UiKitConfiguration
 import org.jetbrains.compose.experimental.uikit.tasks.AbstractComposeIosTask
 import java.io.File
 
 
 fun Project.registerSimulatorTasks(
```

Comment:
Do you know what happen if multiple subproject from the same build are trying to deploy to the same device?

Reply:
This will work's fine.

But at current moment we have another one problem.
If the user changes the existing configuration, the old simulator will remain. We want to make sure that the simulator in this case is recreated with the correct configuration.

Reply:
Kk, thanks for confirming that.

Additional challenge seems to be that these simulators are not visible from XCode (tested with 13.2.1).

Reply:
I also have XCode 13.2.1
This sample will work only with build to mavenLocal of gradle plugin